cuboid_muscle/settings_force.py

```diff
@@ -649,51 +652,51 @@ def callback_function(raw_data):
-    "inputMeshIsGlobal":    True,                         # boundary conditions are specified in global numberings, whereas the mesh is given in local numbering 
-    "solverName":           "linearElasticitySolver",                   # reference to the linear solver
-    "prefactor":            1.0,                                        # prefactor of the lhs, has no effect here
-    "slotName":             "",
-    "dirichletBoundaryConditions": elasticity_dirichlet_bc,             # the Dirichlet boundary conditions that define values for displacements u
-    "dirichletOutputFilename":     None,                                # filename for a vtp file that contains the Dirichlet boundary condition nodes and their values, set to None to disable
-    "neumannBoundaryConditions":   elasticity_neumann_bc,               # Neumann boundary conditions that define traction forces on surfaces of elements
-    "divideNeumannBoundaryConditionValuesByTotalArea": False,           # if the given Neumann boundary condition values under "neumannBoundaryConditions" are total forces instead of surface loads and therefore should be scaled by the surface area of all elements where Neumann BC are applied
+  # "FiniteElementMethod" : {       # linear elasticity finite element method
+  #   "meshName":             "3Dmesh_quadratic",           # mesh with quadratic Lagrange ansatz functions
+  #   "inputMeshIsGlobal":    True,                         # boundary conditions are specified in global numberings, whereas the mesh is given in local numbering 
+  #   "solverName":           "linearElasticitySolver",                   # reference to the linear solver
+  #   "prefactor":            1.0,                                        # prefactor of the lhs, has no effect here
+  #   "slotName":             "",
+  #   "dirichletBoundaryConditions": elasticity_dirichlet_bc,             # the Dirichlet boundary conditions that define values for displacements u
+  #   "dirichletOutputFilename":     None,                                # filename for a vtp file that contains the Dirichlet boundary condition nodes and their values, set to None to disable
+  #   "neumannBoundaryConditions":   elasticity_neumann_bc,               # Neumann boundary conditions that define traction forces on surfaces of elements
+  #   "divideNeumannBoundaryConditionValuesByTotalArea": False,           # if the given Neumann boundary condition values under "neumannBoundaryConditions" are total forces instead of surface loads and therefore should be scaled by the surface area of all elements where Neumann BC are applied
     
-    # material parameters
-    "bulkModulus":          50,     # bulk modulus K, how much incompressible, high -> incompressible, low -> very compressible
-    "shearModulus":         100,      # shear modulus, μ or G, "rigidity", how much shear stress response to shear deformation
+  #   # material parameters
```

Comment:
Remove commented lines

Reply:
Do you mean lines that have been commented out, or also comments?